src/main/java/hudson/plugins/throttleconcurrents/pipeline/ThrottleStep.java

```diff
@@ -0,0 +1,66 @@
+package hudson.plugins.throttleconcurrents.pipeline;
+
+import hudson.Extension;
+import hudson.model.TaskListener;
+import hudson.plugins.throttleconcurrents.ThrottleJobProperty;
+import hudson.util.FormValidation;
+import org.jenkinsci.plugins.workflow.steps.Step;
+import org.jenkinsci.plugins.workflow.steps.StepContext;
+import org.jenkinsci.plugins.workflow.steps.StepDescriptor;
+import org.jenkinsci.plugins.workflow.steps.StepExecution;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.QueryParameter;
+
+import javax.annotation.Nonnull;
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.Set;
+
+public class ThrottleStep extends Step implements Serializable {
+    private String category;
+
+    @DataBoundConstructor
+    public ThrottleStep(@Nonnull String category) {
+        this.category = category;
+    }
+
+    @Nonnull
+    public String getCategory() {
```

Comment:
I would really like to be able to throttle multiple categories as this is our usual scenario for our freestyle jobs.